src/api/controllers/admin/list.js

```diff
@@ -0,0 +1,98 @@
+const { literal, col } = require('sequelize');
+const errorHandler = require('../../utils/errorHandler');
+const { includePay, includeCart } = require('../../utils/customIncludes');
+
+/**
+ * GET /admin/users
+ * {
+ *
+ * }
+ * Response
+ * {
+ *   Users
+ * }
+ * @param {object} infoModel model to query Infos
+ * @param {object} teamModel model to query Infos
+ * @param {object} tournamentModel model to query Infos
+ */
+const List = (userModel, teamModel, tournamentModel, cartModel, cartItemModel, itemModel) => async (req, res) => {
+  const page = req.query.page || 0;
+  const pageSize = 25;
+  const offset = page * pageSize;
+  const limit = pageSize;
+  const filterTournament = req.query.tournamentId === 'all' ? undefined : req.query.tournamentId;
+  const filterScan = req.query.scan === 'all' ? undefined : req.query.scan;
+  const customScan = filterScan ? ` AND user.scanned IS ${filterScan}` : '';
+  let customWhere = undefined;
+  switch (req.query.status) {
+    case 'paid':
+      customWhere = literal(`forUser.id IS NOT NULL ${customScan}`);

```

Comment:
La def de la fonction de literal sur Sequelize c'est `Creates an object representing a literal, i.e. something that will not be escaped.`
J'espère que par `that will not be escaped` ils ne parlent pas d'échappement d’apostrophes, car sinon Injection SQL...

Reply:
Si totalement
@florianess faut trouver un autre moyen de faire ça, là c'est les injections SQL c'est critique